kuma/wiki/tests/test_forms.py

```diff
@@ -537,8 +537,8 @@ def test_akismet_error(self, mock_requests):
     @pytest.mark.spam
     @requests_mock.mock()
     def test_akismet_spam_training(self, mock_requests):
-        flag, created = Flag.objects.get_or_create(name=SPAM_TRAINING_FLAG)
-        flag.users.add(self.testuser)
+        switch, created = Switch.objects.get_or_create(name=SPAM_TRAINING_SWITCH)
+        switch.users.add(self.testuser)
```

Comment:
@jwhitlock Not sure how to handle this as the `Switch` object does not have a user property.

Reply:
So, from the code I see that "Switches are active, or inactive, globally." so, perhaps I should just set active to `True` and remove line 541?

Reply:
This code is being overly cautious, and can be replaced with:

```
Switch.objects.create(name=SPAM_TRAINING_SWITCH, active=True)
```

Same for ``test_akismet_error_training`` below.